src/main/java/com/vi/appointmentservice/api/controller/AskerController.java

```diff
@@ -24,8 +25,21 @@ public class AskerController implements AskersApi {
   private final @NonNull AskerFacade askerFacade;
 
   @Override
-  public ResponseEntity<List<CalcomBooking>> getAllBookingsOfAsker(String askerId) {
-    return new ResponseEntity<>(askerFacade.getAllBookingsOfAskerHandler(askerId), HttpStatus.OK);
+  public ResponseEntity<List<CalcomBooking>> getAllBookingsOfAsker(String askerId, String status) {
```

Comment:
advice seeker can only see upcoming appointments. we this implementation, we have to change the frontend and we need to add more tests, even though we don't need this functionality now.

Reply:
Okay, I missunderstood when you said the asker API should work like the consultant API.

Reply:
@adnanalicic Please re-review